test/parallel/test-buffer-tostring.js

```diff
@@ -23,3 +23,11 @@ encodings
   assert.strictEqual(Buffer.from('666f6f', encoding).toString(encoding),
                      '666f6f');
 });
+
+// invalid encodings
```

Comment:
I'd prefer the comment to start with a capital letter, but maybe it's just me, idk :)